### DISCUSSION TITLE
Can flagd be using a serverless service (aws lambda, ...) ?

### DISCUSSION THREAD

**Discussion by victorlevasseur** [open-feature/flagd#1361] [Q&A] [answered]
Hi,

I would like to know if there's any issue that may arise if I deploy flagd on a serverless service like AWS Lambda.

I saw that some OpenFeature clients support event streaming, will it affect it?

Thanks.

**Comment by beeme1mr** [comment, marked as answer]
I don't think flagd would work well in a serviceless environment but you could try something like AWS Fargate.

What are your goals? I can provide some recommendations depending on what you're hoping to achieve. Thanks!

**Reply by victorlevasseur** [reply]
It was just to optimize the cost, but Fargate is indeed a good solution
